src/applications/disability-benefits/all-claims/tests/pages/contactInformation.unit.spec.jsx

```diff
@@ -240,7 +240,9 @@ describe('Disability benefits 526EZ contact information', () => {
           },
           'view:hasForwardingAddress': true,
           forwardingAddress: {
-            effectiveDate: '2019-01-01',
+            effectiveDate: {
+              from: '2019-01-01',
```

Comment:
Should there be a test written for `effectiveDate.to`? Maybe that it displays as expected? :man_shrugging:

Reply:
Added some tests around validating the effective dates. And now it uses the common `dateRangeUI` uiSchema so I'm going to assume that's tested elsewhere.